homeassistant/components/fritz/device_tracker.py

```diff
@@ -79,6 +79,14 @@ def get_device_name(self, device):
             return None
         return ret
 
+    def get_extra_attributes(self, device):
+        """Return the name of the given device or None if is not known."""
+        ip_device = self.fritz_box.get_specific_host_entry(device).get("NewIPAddress")
+
+        if ip_device == {}:
```

Comment:
```py
if not ip:
```

Reply:
thankyou for your feedback!
I changed it in the recent commit